Documents/API.md

```diff
@@ -275,7 +289,8 @@ All endpoints in the `/institutions` namespace require two headers (see "Authori
         {
             "edits": [
                 {
-                   "edit": "q007",
+                   "edit": "Q007",
+                   "description: "Description of Q007",
```

Comment:
You have the colon and quotes a bit mixed up.

`"description: "Description of Q007",`

should be 

`"description": "Description of Q007",`